dradis/dradis-ce
# General presenter

### Spec
Currently, we have an ActivityPresenter that we use when we display activities. With the comments and the notifications coming into play, we can use some of the methods from the ActivityPresenter to display these items such as the avatar, partial, etc. to avoid reinventing the wheel.

#### Proposed Solution
We need to create a general/higher Presenter that the other presenters can inherit from with the reusable methods.

Additionally, to make the general Presenter code be more general we need to associate Activities with users (now that we have the proper User DB) instead of having a user email attribute.

### How to test
This is a refactor PR, but we need to make sure that the Recent Activities view is unchanged.
